tests/res_diff_tests/test_fix_read_compare_errors.ion

```diff
@@ -0,0 +1,22 @@
+{
+ bad: 	{
+	"test": {
+        'ion-java_1': {
+            result: FAIL,
+             read_compare: {
+                errors: ErrorReport::[
+                    {
+                      message: "test",
+                    }
+                ]
+             }
+        },
+        'ion-java_2': {
+            result:FAIL
```

Comment:
If the result field passes or not is depending on the other four phases (read, read_compare, write, write_compare). It's FAIL as long as we write anything to report for these four phases. So the pass/fail depends on if the implementations behavior same rather than if they face errors.

So even though it faces a parsing failure, as long as all implementations has the same incomplete event stream output, it's still treated as PASS.

Sure, Now we check two things: return value and output. return value should be a expected value (0 for pass and non-zero for fail) and output should be same as expected output.